test/controllers/ReasonControllerSpec.scala

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import akka.stream.Materializer
+import com.google.inject.Inject
+import config.FrontendAppConfig
+import forms.ReasonFormProvider
+import identifiers.TypedIdentifier
+import models.NormalMode
+import models.requests.DataRequest
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.{MustMatchers, OptionValues, WordSpec}
+import play.api.i18n.MessagesApi
+import play.api.inject.bind
+import play.api.mvc.{AnyContent, Call, Request, Result}
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import services.{FakeUserAnswersService, UserAnswersService}
+import uk.gov.hmrc.domain.PsaId
+import utils.{FakeNavigator, Navigator, UserAnswers}
+import viewmodels.ReasonViewModel
+import views.html.reason
+
+import scala.concurrent.Future
+
+class ReasonControllerSpec extends WordSpec with MustMatchers with OptionValues with ScalaFutures {
+
+  import ReasonControllerSpec._
+
+  val viewmodel = ReasonViewModel(
+    postCall = Call("GET", "www.example.com"),
+    title = "title",
+    heading = "heading"
+  )
+
+//  "get" must {
```

Comment:
I know you're testing the NoCompanyUTRController, do we need the commented tests in here or can we get rid of them?